com.unity.render-pipelines.high-definition/Editor/Material/UIBlocks/SurfaceOptionUIBlock.cs

```diff
@@ -698,6 +699,14 @@ void SurfaceTypePopup()
                         renderQueueType = HDRenderQueue.ConvertFromTransparentRenderQueue(newRenderQueueTransparentType);
                         renderQueue = HDRenderQueue.ChangeType(renderQueueType, offset: (int)transparentSortPriority.floatValue);
                     }
+                    if (renderQueueTransparentType == HDRenderQueue.TransparentRenderQueue.LowResolution)
+                    {
+                        if (HDRenderPipeline.currentPipeline != null
+                            && !HDRenderPipeline.currentPipeline.currentPlatformRenderPipelineSettings.lowresTransparentSettings.enabled)
+                        {
+                            EditorGUILayout.HelpBox(Styles.lowResTransparencyNotSupportedText, MessageType.Info);
```

Comment:
While I like this, one thing to keep in mind is that the asset used during authoring is not necessarily the asset that is used in the project on a given platform. 

I think this is the reason why we don't do this for similar things (e.g. subsurface scattering is a very similar situation)

Reply:
I do like the idea though, worth pinging UX?

Reply:
Yep, pinged maddalena, she liked it as well